### DISCUSSION TITLE
Added support for Human (face) running in a serverless environment

### DISCUSSION THREAD

**Discussion by nuxy** [vladmandic/human#396] [Show and tell]
I wrote the following package which allows you to run [TensorFlow](https://github.com/tensorflow/tfjs) and [Human](https://github.com/vladmandic/human) in a serverless [AWS Lambda](https://aws.amazon.com/lambda) container environment. The motivation behind this was to avoid having to run a dedicated server (a cost in itself) that would be periodically used.  Pay for play.

While this package was written to meet my specific use-case _it can be extended pretty easily_.

https://github.com/nuxy/human-face-detection

With that said, thank you for supporting the [face-api](https://github.com/vladmandic/face-api) and creating this package.  Very helpful.

Now if I can only get it to detect :cat: faces.

**Comment by vladmandic** [comment]
very nice - thanks for sharing!
